src/opts.rs

```diff
@@ -32,6 +32,16 @@ pub enum ActionClientOnly {
     /// Print and watch the eww logs
     #[structopt(name = "logs")]
     Logs,
+    /// Edit the configuration files
+    #[structopt(name = "edit")]
+    Edit {
+        /// Editor to use
+        #[structopt(name = "editor", short, long)]
+        editor_arg: Option<String>,
+        /// What file to edit (the main xml file or the main scss file) accepted values are xml or scss
```

Comment:
A much nicer implementation of this would be to use an enum type with the two options.
As we'll likely be waiting for multi-file config to be finished, using strings will be preferrable if these are done... thus i guess using strings is fine here.

Reply:
wow very amazing suggestion then, if we're waiting anyways for his PR